prediction_market_agent_tooling/markets/omen/omen_subgraph_handler.py

```diff
@@ -0,0 +1,302 @@
+import sys
+import typing as t
+from datetime import datetime
+
+import subgrounds.subgraph
+from eth_typing import ChecksumAddress
+from hexbytes import HexBytes
+from subgrounds import FieldPath, Subgrounds
+
+from prediction_market_agent_tooling.gtypes import HexAddress
+from prediction_market_agent_tooling.markets.agent_market import FilterBy, SortBy
+from prediction_market_agent_tooling.markets.omen.data_models import (
+    OMEN_FALSE_OUTCOME,
+    OMEN_TRUE_OUTCOME,
+    OmenBet,
+    OmenMarket,
+    OmenUserPosition,
+    RealityAnswer,
+)
+from prediction_market_agent_tooling.tools.utils import to_int_timestamp, utcnow
+
+
+class OmenSubgraphHandler:
+    """
+    Class responsible for handling interactions with Omen subgraphs (trades, conditionalTokens).
+    """
+
+    OMEN_TRADES_SUBGRAPH = "https://api.thegraph.com/subgraphs/name/protofire/omen-xdai"
+    CONDITIONAL_TOKENS_SUBGRAPH = (
+        "https://api.thegraph.com/subgraphs/name/gnosis/conditional-tokens-gc"
+    )
+    REALITYETH_GRAPH_URL = (
+        "https://api.thegraph.com/subgraphs/name/realityeth/realityeth-gnosis"
+    )
+
+    # We define here as str for easier filtering.
+    INVALID_ANSWER_STR = HexBytes(
```

Comment:
It's called `_STR`, but the type is `bytes` (HexBytes). Before, this was written as 

```
INVALID_ANSWER = 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
```

so the type was `string` and not `bytes`.

```suggestion
    INVALID_ANSWER_STR = HexStr(
```

Could you double check or just correct me if you know already, which one is correct, please?

We can either remove `_STR` suffix or change the type.